lib/formtastic/inputs/base/validations.rb

```diff
@@ -208,8 +204,22 @@ def readonly_attribute?
         def readonly_from_options?
           options[:input_html] && options[:input_html][:readonly]
         end
+
+        private
+
+        # Loosely based on
+        # https://github.com/rails/rails/blob/5-2-stable/activemodel/lib/active_model/validations/numericality.rb#L54-L59
+        def option_value(option, object)
+          case option
+          when Symbol
```

Comment:
I think both can be unified to:

```ruby
option.to_proc.call(option)
```

Reply:
But this might behave different than the rails validator on private methods. So better to keep it like this.